services/block_service.go

```diff
@@ -102,19 +102,18 @@ func containsTopic(log *EthTypes.Log, topic string) bool {
 	return false
 }
 
-func (s *BlockAPIService) populateTransaction(
-	_ context.Context,
+func (s *BlockAPIService) PopulateTransaction(
+	ctx context.Context,
 	tx *client.LoadedTransaction,
 ) (*RosettaTypes.Transaction, error) {
-	var ops []*RosettaTypes.Operation
+	// var ops []*RosettaTypes.Operation
```

Comment:
make sure to remove a comment if not needed or add an explanation on why this is commented and can be used later